### DISCUSSION TITLE
PocketBase Sync: Synchronize Individual Collection Schema & Data between PocketBase Instances

### DISCUSSION THREAD

**Discussion by Mashiane** [pocketbase/pocketbase#2419] [Show and tell]
Hi there

I usually develop my and test my PocketBase Applications locally and when I need to present them, move the schemas to PocketHost.
With PocketHost, one is able to create different instances of databases whilst locally on a PC, you just have 1 PocketBase instance.

Currently, if you Export and Import collections, you are faced with deleting or even overwriting content. I wanted a tool that could help me sync any schema of choice between PocketHost and a locally running PocketBase instance.

This is developed using TailwindCSS and DaisyUI. Check it out.

https://candid-daffodil-9a4d11.netlify.app/

https://youtu.be/RNdVqBs7Xh0

**Comment by Mashiane** [comment]
You can now read about this on Medium

https://medium.com/javascript-in-plain-english/how-to-sync-individual-pocketbase-collection-schemas-and-data-d7db1647d730

**Reply by Mashiane** [reply]
Update: Here is the github repo, https://github.com/Mashiane/SithasoDaisy-PocketHost-Sync

**Comment by sergioponguta** [comment]
No github repo for the tool?
